src/main/kotlin/no/nav/syfo/sykmelding/serviceuser/api/SykmeldingServiceuserApi_getAllByFnr.kt

```diff
@@ -0,0 +1,31 @@
+package no.nav.syfo.sykmelding.serviceuser.api
+
+import io.ktor.application.call
+import io.ktor.http.ContentType
+import io.ktor.http.HttpStatusCode
+import io.ktor.response.respond
+import io.ktor.routing.Route
+import io.ktor.routing.accept
+import io.ktor.routing.get
+import io.ktor.routing.route
+import no.nav.syfo.log
+import no.nav.syfo.sykmelding.service.SykmeldingerService
+import java.time.LocalDate
+
+fun Route.registrerSykmeldingServiceuserApiV1_getAllByFnr(sykmeldingerService: SykmeldingerService) {
+    route("api/v1/sykmelding") {
```

Comment:
Kunne vi ha inkludert dette i registrerSykmeldingServiceuserApiV1 som finnes fra før..? Altså, to routes i samme metoden? Det slo meg først nå at det kanskje er like greit siden det i praksis er to ulike operasjoner på samme api :)

Reply:
Haha, ja, det er en mye bedre idé :)